polar-core/src/polar.rs

```diff
@@ -296,6 +298,17 @@ impl Polar {
     pub fn next_message(&self) -> Option<Message> {
         self.messages.next()
     }
+
+    /// Load the Polar roles policy idempotently.
+    pub fn enable_roles(&self) -> PolarResult<()> {
+        match self.load(ROLES_POLICY, Some("Built-in Polar Roles Policy".to_owned())) {
+            Err(error::PolarError {
+                kind: error::ErrorKind::Runtime(error::RuntimeError::FileLoading { .. }),
+                ..
+            }) => Ok(()),
```

Comment:
Why are we returning `Ok(())` on this error?

Reply:
If it's to ignore duplicate file errors--is that the only case which could produce the `FileLoading` error, or would we potentially be ignoring any valid errors?

Reply:
> If it's to ignore duplicate file errors

yep

> is that the only case which could produce the FileLoading error

yep

> would we potentially be ignoring any valid errors?

nope